packages/x-date-pickers/src/PickersTextField/PickersInputBase/PickersInputBase.types.ts

```diff
@@ -67,4 +68,5 @@ export interface PickersInputBaseProps
     root?: any;
     input?: any;
   };
+  'data-multiinput'?: string;
```

Comment:
Nitpick, to follow Base UI conventions:

```suggestion
  'data-multi-input'?: string;
```

Reply:
Good suggestion. For some reason, I thought that the attribute name had to be a single word. 🙈 🤷